MAVProxy/modules/mavproxy_soar.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+'''
+Soaring Module
+Ryan Friedman
+
+This module displays the estimated soaring thermals on the map.
+A circle is drawn with the estimated radius from ArduSoar's estimated thermal location.
+
+AP_FLAKE8_CLEAN
+'''
+
+from MAVProxy.modules.lib import mp_module, mp_util
+from MAVProxy.modules.mavproxy_map import mp_slipmap
+
+
+class soar(mp_module.MPModule):
+    def __init__(self, mpstate):
+        """Initialise module"""
+        super(soar, self).__init__(mpstate, "soar", "")
+
+    def mavlink_packet(self, m):
```

Comment:
I sgugest dropping a timeout in the top of here to clear the circle if you haven't got a message in a while.

Reply:
Nice, yea good idea. Added!